ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js.txt

```diff
@@ -1 +1,3 @@
-js/container.js
\ No newline at end of file
+js/container.js
+js/tiny-slider.js
```

Comment:
@Rajamouli-hcl : this should be part of clientlib site

Reply:
Updated as per comments!